code/modules/mob/new_player/new_player.dm

```diff
@@ -312,7 +312,7 @@
 		to_chat(src, alert("[rank] is not available. Please try another."))
 		return 0
 	var/datum/job/thisjob = SSjobs.GetJob(rank)
-	if(thisjob.barred_by_disability(client))
+	if(thisjob.barred_by_disability(client) || thisjob.barred_by_missing_limbs(client))
```

Comment:
The same applies here but for another reason. It's probably better to give the user specific details as to why they can't play.

Reply:
Fixed in [c3edc2e](https://github.com/ParadiseSS13/Paradise/pull/20338/commits/c3edc2ec34e9c3f1fd40123a568057f9df85c80f)